### PR TITLE
[MOB-28] Install cypress and first test 

### PR DESCRIPTION
**Pull request by YagoAres**
## Pull Request Checklist

- [x] **Title**
- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-28)**
- [x] **Add PR url to US**

Added cypress dependencies and their installation with a test example 

## Testing Guide

For run cypress clone the repository and in terminal (pointing to the downloaded repository):

1. 'npm install' to install all the dependencies required 

```
npm install
```
2. cd to test-e2e folder
3. And run cypress with 'npx cypress open'

```
npx cypress open
```

### CODE REVIEW

File: test-e2e/cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
```
Comment (LetiRocha) [inline test-e2e/cypress/support/commands.js:2]: Don't add classes with comments only

File: test-e2e/cypress/support/e2e.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (LetiRocha) [inline test-e2e/cypress/support/e2e.js:1]: same, don't add classed with only commented code, cause this only includes a reference to the previous one that is only commented code

File: test-e2e/cypress/fixtures/example.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (LetiRocha) [inline test-e2e/cypress/fixtures/example.json:1]: this is not being used anywhere either

**Review by LetiRocha — CHANGES_REQUESTED** [review]

**Review by LetiRocha — CHANGES_REQUESTED** [review]
Please rename the file of the initial test to follow the naming standar in the rest of the project

File: test-e2e/cypress.config.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+module.exports = defineConfig({
+  viewportHeight: 1080, //Resolución de la pantalla
+  viewportWidth: 1920,
+
+  e2e: {
+    setupNodeEvents(on, config) {
+
+    },
+
+    baseUrl: 'https://qa.esmorga.canarte.org/', //URL base de la aplicación
```
Comment (LetiRocha) [inline test-e2e/cypress.config.js:12]: I don't think that comment is needed

File: test-e2e/cypress.config.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    specPattern: 'cypress/e2e/**/*.spec.{js,jsx,ts,tsx}', //Patrón de búsqueda de archivos de prueba
+    supportFile: false //Archivo de soporte no necesario
```
Comment (LetiRocha) [inline test-e2e/cypress.config.js:14]: comments should be in english as the rest of the code

File: test-e2e/cypress/e2e/1test.spec.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+describe('Testing is all ok', () => {
+  //Test de prueba para verificar que todo está funcionando correctamente
```
Comment (LetiRocha) [inline test-e2e/cypress/e2e/1test.spec.js:2]: This comments doesn't bring any value

**Review by LetiRocha — APPROVED** [review]

**Review by LetiRocha — APPROVED** [review]

**Review by YagoAccenture — APPROVED** [review]

**Review by PabloCanarteSol — APPROVED** [review]

**Pull request merged**